src/game/game.cpp

```diff
@@ -528,12 +528,32 @@ namespace game
 
     bool inzoom()
     {
-        if(lastzoom && (zooming || lastmillis-lastzoom <= W(focus->weapselect, cookzoom)))
-            return true;
-        return false;
+        return lastzoom && (zooming || lastmillis - lastzoom <= W(focus->weapselect, cookzoom));
     }
     ICOMMAND(0, iszooming, "", (), intret(inzoom() ? 1 : 0));
 
+    // Get the progress of the zoom in the interval [0, 1], where 0 is no zoom and 1 is full zoom.
+    float zoomratio(gameent *d)
```

Comment:
AFAIR we decided to use snake_case for new variables and functions. Please adjust.